receiver/sqlserverreceiver/metadata.yaml

```diff
@@ -177,6 +177,60 @@ metrics:
       value_type: double
     attributes: [physical_filename, logical_filename, file_type]
     extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.database.io.write_latency:
+    enabled: false
+    description: Total time that the users waited for writes.
+    unit: "s"
+    sum:
+      monotonic: true
+      aggregation_temporality: cumulative
+      value_type: double
+    attributes: [physical_filename, logical_filename, file_type]
+    extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.database.io.reads:
+    enabled: false
+    description: Number of reads issued on the file.
+    unit: "{reads}"
+    sum:
+      monotonic: true
+      aggregation_temporality: cumulative
+      value_type: int
+      input_type: string
+    attributes: [physical_filename, logical_filename, file_type]
+    extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.database.io.writes:
+    enabled: false
+    description: Number of writes made on this file.
+    unit: "{writes}"
+    sum:
+      monotonic: true
+      aggregation_temporality: cumulative
+      value_type: int
+      input_type: string
+    attributes: [physical_filename, logical_filename, file_type]
+    extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.database.io.read:
```

Comment:
I'd appreciate feedback on metric names and format here, to ensure they match [semantic conventions](https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-naming). I'm not sure if these `read/write` metrics should be converted to a single metric for each, with `read` vs `write` attributes, or leave them like this.

Reply:
I think we should have `read`/`write` as part of `direction` attribute. See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/pagingscraper/metadata.yaml as an example.

`sqlserver.database.io.writes` is hard to distinguish from `sqlserver.database.io.write`. We can use `sqlserver.database.io` and `sqlserver.database.operations` instead

Reply:
If `sqlserver.database.operations ` isn't specific enough, we can do `sqlserver.database.io.operations` and `sqlserver.database.io.bytes`

Reply:
I agree they weren't easy to distinguish, works for me.

I've updated to `sqlserver.database.io` and `sqlserver.database.operations` for now, pending feed back from @djaglowski.